tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,109 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/Plex.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --device=all
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download:create
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
+  - --filesystem=home:ro
```

Comment:
I've explained in the PR text why the non-standard ones are needed. If you have any further questions, let me know. But I wasn't aware this could be a point of contention since there are other closed source applications on flathub.

Reply:
We'll review the permissions used, but there might still be a good number of them that we'll need for the application to function.

Reply:
Yeah, the context of use being basically 'an operating system UI' is going to need more things than you'd otherwise want. Worth checking for sure though.

Reply:
I've removed a number of permissions. See the updated post for explanations for the remaining ones.